src/common/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_library(common STATIC common.c)
```

Comment:
Because it's used multiple times by multiple targets and so I'm avoiding re-compiling the source file everytime?